nbdev/qmd.py

```diff
@@ -72,3 +72,18 @@ def tbl_sep(sizes:int|list=3  # List of column sizes, or single `int` if all siz
     "Create a markdown table separator with relative column size `sizes`"
     if isinstance(sizes,int): sizes = [3]*sizes
     return tbl_row('-'*s for s in sizes)
+
+# %% ../nbs/api/qmd.ipynb 10
+def _install_nbdev():
```

Comment:
@hamelsmu @jph00 I'd like to hear your take on this

Reply:
More generally, what do you think about moving repeated doc snippets into internal functions like this? This PR uses `_install_nbdev` in git friendly jupyter, tutorial, and pre commit doc pages.

Reply:
lgtm!